2023-MRC-BSU-and-UKHSA/getting-started.md

```diff
@@ -31,31 +24,17 @@ At the end of this process you should have access to some way of getting you int
 julia> 
 ```
 
-For example, on a Linux device this is generally achieved by running the command `julia` or the executable `/wherever/you/installed/julia/bin/julia`.
-
-(If the version is slightly different, e.g. 1.9.2, that is no problem; the important bit is that you're on a version ≥1.9)
+For example, on Linux and mac this is generally achieved by running the command `julia` or the executable `/wherever/you/installed/julia/bin/julia` at the terminal.
 
-## "But how do I *write* Julia code?!" - Setting up an editor
+# Editor Choice
 
-As mentioned earlier, at certain points during these workshops you're likely to find yourself in a position where you even need to *write* some Julia code. I know, this is all very demanding.
-
-How and where and with what you want to write these chunks of text is of course completely up to yourself.
-
-- If you live and die by Notepad++, then you're welcome to use Notepad++.
-- If you're one of those wizards who doesn't have to spend 30 minutes trying to exit a file you opened in `vim`, then maybe you should use `vim`.
-- Or if you're part of the opposition of the group of people described in the previous line, i.e. an `emacs` user, then you might want to use `emacs`.
-
-**But most (sane) people will probably prefer something like:**
-1. a proper IDE, in which case [Visual Studio (VS) Code](https://code.visualstudio.com/) with [the Julia extensions](https://www.julia-vscode.org/) is a great option, or
+You are free to use your preferred editor (emacs, vims, sublime, notepad++ etc), however, we generally recommend either
+1. [Visual Studio (VS) Code](https://code.visualstudio.com/) with [the Julia extensions](https://www.julia-vscode.org/), or
 2. [Jupyter notebooks](https://jupyter.org/), which is easily supported by simply installing the Julia package `IJulia`; once this is installed you should find a `julia` kernel available in your notebook.[^1]
 
 The latter option might be more familiar if you're coming from a scientific Python background, but VS Code is generally the recommended option.
-
 Amongst us speakers we're familiar with Jupyter notebooks, Will and Xianda are VS Code users, and Tor is a (grumpy) `emacs` user, hence we should be able to provide _some_ support with a few different setups.
 
-Note that we expect everyone to act civil during the workshops, regardless of the differences in choices of editor people make. 
-In particular, any criticism of `emacs` will be met by loud boos and shaking of fists from Tor in an attempt to quell potential conflicts.
```

Comment:
Amusing, but only people who are familiar with the emacs vs vim debate will get this